src/Stats.js

```diff
@@ -87,13 +150,37 @@ function LogTable(props) {
 }
   
 function TableRow(props) {
+  const [standardType, setStandardType] = useContext(StandardTypeContext);
+
   return (
     <Table.Row>
       <Table.Cell>
         <Header as='h4' textAlign='center'>{props.exerciseName}</Header>
       </Table.Cell>
-      <Table.Cell textAlign='center'>{props.tested1RM}</Table.Cell>
+      <Table.Cell textAlign='center' >{props.tested1RM}</Table.Cell>
       <Table.Cell textAlign='center'>{props.estimated1RM}</Table.Cell>
+      <Table.Cell textAlign='center'>{standardType === 'Estimated Level' ? <h1 style={{color: props.standard[0][1]}}>{props.standard[0][0]}</h1> : <h1 style={{color: props.standard[1][1]}}>{props.standard[1][0]}</h1>}</Table.Cell>
     </Table.Row>
   )
-}
\ No newline at end of file
+}
+
+function calculateUserWorldRecord(bw, gender, exerciseName) {
+
+  if (gender === 'male') {
```

Comment:
Use constants for this. src/Constants.js and  

export const MALE = "male";
export const FEMALE = "female"; 

This prevents you from using "Male" in a different file and then values don't match